carbonmark-api/src/models/Project.model.ts

```diff
@@ -1,34 +1,46 @@
 import { Static, Type } from "@sinclair/typebox";
 import { ListingModel } from "./Listing.model";
 import { MethodologyModel } from "./Methodology.model";
+import { TokenPriceModel } from "./TokenPrice.model";
 import { GeoJSONPointModel, Nullable } from "./Utility.model";
 
 export const ProjectModel = Type.Object({
   description: Nullable(Type.String()),
+  long_description: Nullable(Type.String()),
   short_description: Nullable(Type.String()),
   key: Type.String(),
   projectID: Type.String(),
   name: Type.String(),
-  methodologies: Type.Array(Nullable(MethodologyModel)),
+  methodologies: Type.Array(MethodologyModel),
   location: Nullable(GeoJSONPointModel, {
     description: "A GeoJSON Point feature.",
   }),
+  url: Nullable(Type.String()),
   vintage: Type.String(),
-  creditTokenAddress: Type.String(),
+  creditTokenAddress: Type.String({
+    description: "⚠️Deprecated. Project may have multiple token addresses.",
+  }),
   registry: Type.String(),
   updatedAt: Type.String(),
-  country: Type.Object({ id: Type.String() }),
+  country: Type.String(),
   region: Type.String(),
   price: Type.String(),
+  prices: Type.Array(TokenPriceModel),
+  stats: Type.Object({
+    totalBridged: Type.Number(),
+    totalRetired: Type.Number(),
+    totalSupply: Type.Number(),
+  }),
```

Comment:
I see we've aligned this model with the `DetailedProjectModel` was this so you could consolidate the `buildProjectEntry` function?

The original intent for having the two models was that some of this data is not required in the case of `/projects` given that the payload size is getting prohibitively large we should only provide the data that is necessary.

**Staging**
<img width="761" alt="Screenshot 2024-01-08 at 11 19 05 am" src="https://github.com/KlimaDAO/klimadao/assets/7104689/9d50e46e-617e-405f-a095-2233f10d6376">

**2020_hide_dust_api**
<img width="772" alt="Screenshot 2024-01-08 at 11 21 35 am" src="https://github.com/KlimaDAO/klimadao/assets/7104689/7d0bbe73-796c-4cdd-8451-bd7b39338bfa">

It looks like we might have doubled that payload size?

I agree that a seperate model just to cater for a specific use case is not great however I'm not sure if the solution here is to add paging or even graphql so that the consumer can decide what data is necessary for their use case..

Thoughts?

@Atmosfearful @biwano